helpdesk_mgmt_ticket_assignment_massive/wizards/assignement_massive.py

```diff
@@ -0,0 +1,32 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models, api
+
+
+class AssignementMassive(models.TransientModel):
+    _name = 'helpdesk.ticket.assignement.massive.wizard'
+
+    ticket_ids = fields.Many2many(
+        'helpdesk.ticket',
+        string="Ticket",
+        readonly=True
+    )
+    user_id = fields.Many2one('res.users', string='Assigned user')
+    stage_id = fields.Many2one('helpdesk.ticket.stage', string="Stage")
+
+    @api.depends('ticket_ids, stage_id')
+    def assigned_user(self):
+        for ticket in self.ticket_ids:
+            if(self.user_id):
+                ticket.write({
+                    'user_id': self.user_id.id
+                })
+            if(self.stage_id):
```

Comment:
This code doesn't cover to corner cases:
- If `user_id` is not filled, the tickets aren't modified. But, if we want tu unassign those tickets?
- If any ticket has a `team_id` incompatible with the selected `user_id`, an error should be fired and proccss stopped.